src/pages/workspace/members/WorkspaceMemberDetailsPage.tsx

```diff
@@ -128,14 +130,52 @@ function WorkspaceMemberDetailsPage({personalDetails, policy, route}: WorkspaceM
 
     const confirmModalPrompt = useMemo(() => {
         const isApprover = isApproverUserAction(policy, memberLogin);
+        const isTechnicalContact = policy?.technicalContact === details?.login;
+        const exporters = [
+            policy?.connections?.intacct?.config?.export?.exporter,
+            policy?.connections?.quickbooksDesktop?.config?.export?.exporter,
+            policy?.connections?.quickbooksOnline?.config?.export?.exporter,
+            policy?.connections?.xero?.config?.export?.exporter,
+            policy?.connections?.netsuite?.options.config.exporter,
```

Comment:
👋 Hey, coming from https://github.com/Expensify/App/issues/74903.
This has caused a crash when trying to connect to NetSuite with incorrect credentials, we were missing optional chaining for netsuite exporter

> policy?.connections?.netsuite?.options?.config?.exporter,